src/nvim/strings.h

```diff
@@ -65,3 +62,5 @@ static inline char *strappend(char *const dst, const char *const src)
 #  define STRNICMP(d, s, n) vim_strnicmp((char *)(d), (char *)(s), (size_t)(n))
 # endif
 #endif
+
+#define kv_printf(v, ...) kv_do_printf(&(v), __VA_ARGS__)
```

Comment:
I moved this here from `klib/kvec.h` since `kv_do_printf` is declared in `strings.h.generated.h`. So any file that wants to use `kv_printf` needs to depend on `strings.h`, not (just) `kvec.h`.